package.json

```diff
@@ -12,14 +12,9 @@
   "types": "dist/types/oidc-client-ts.d.ts",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/types/oidc-client-ts.d.ts",
-        "default": "./dist/esm/oidc-client-ts.js"
-      },
-      "require": {
-        "types": "./dist/types/oidc-client-ts.d.ts",
-        "default": "./dist/umd/oidc-client-ts.js"
-      }
+      "import": "./dist/esm/oidc-client-ts.js",
+      "require": "./dist/umd/oidc-client-ts.js",
+      "types": "./dist/types/oidc-client-ts.d.ts"
```

Comment:
Make sure that `"types"` is listed first. TS will evaluate the "import" condition for cases where the the `.d.ts` is stored as a sibling

![image](https://user-images.githubusercontent.com/4993980/173466228-a6bbe049-0c49-43a1-82ce-37ca78e88722.png)

Reply:
Done